server/src/internalClusterTest/java/org/elasticsearch/search/aggregations/pipeline/AvgBucketIT.java

```diff
@@ -45,6 +45,9 @@
     static int numValueBuckets;
     static long[] valueCounts;
 
+    static String histoName;
+    static String termsName;
+
```

Comment:
I have only do it for one test but maybe we should do it for all of them for consistency? Maybe a follow up PR?

Reply:
Yeah, I think that'd be good to do in a follow up PR.